web-components/index.html

```diff
@@ -16,7 +16,23 @@
 </head>
 
 <body>
-  <usa-banner></usa-banner>
+  <usa-banner label="My custom aria label"></usa-banner>
+  <usa-banner tld="mil"></usa-banner>
+  <usa-banner lang="es"></usa-banner>
+  <usa-banner lang="yor" tld="mil" label="Custom Aria label">
+  <span slot="banner-text">Custom banner text I've created</span>
+  <span slot="banner-action">Learn more</span>
+  <span slot="domain-heading">Domain info</span>
+  <span slot="domain-text">
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Hic, libero.
+  </span>
+  <span slot="https-heading">HTTPS info</span>
+  <span slot="https-text">
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto, exercitationem.
+  </span>
+
+</usa-banner>
+
```

Comment:
![image](https://github.com/uswds/uswds-next/assets/3385219/b5247a53-2272-49cd-9534-a9f47ccfb3e8)

Reply:
Cool to see the level of customization we can accomplish with slots and that data object!


I'm curious, any idea what users who have customized the banner further would be able to accomplish? Things like removing the icons, or [Vote.gov](https://vote.gov/)'s example where they added a language selector to their banner

Reply:
That could be a future enhancement. Right now there's currently no available generic `<slot>` element.